docs/template_plugin/tests/functional/op_reference/comparison.hpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <vector>
+
+#include "base_reference_test.hpp"
+#include "ngraph_functions/builders.hpp"
+
+namespace reference_tests {
+namespace ComparisonOpsRefTestDefinitions {
+
+struct RefComparisonParams {
+    ngraph::helpers::ComparisonTypes compType;
+    Tensor input1;
+    Tensor input2;
+    Tensor expected;
+};
+
+struct Builder : ParamsBuilder<RefComparisonParams> {
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, compType);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, input1);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, input2);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, expected);
+};
+
+class ReferenceComparisonLayerTest : public testing::TestWithParam<RefComparisonParams>, public CommonReferenceTest {
```

Comment:
Should you remove some backend tests?

Reply:
Yes, but as I don't see anything against temporarily having doubled tests for those ops (less, equal) I would like to do that in a separate PR as this one is already 7 days in review.

Reply:
@blesniewski Why cannot we do such things immediately? In order to don't return to previous activities.

Reply:
In this case the reason was constant issues with the CI not wanting to pass and time in the sprint running out. I've removed some backend tests.